backend/db/models/song.js

```diff
@@ -3,6 +3,11 @@ const { Model } = require("sequelize");
 module.exports = (sequelize, DataTypes) => {
   class Song extends Model {
     static associate(models) {
+      const playlistsSongMap = {
```

Comment:
Doesnt look like you are using the mapping object, if you aren't using it you should delete it, if you need it then make sure you pass it in as an argument to the association function.